test.js

```diff
@@ -663,12 +663,31 @@ test('batch.list()', async (t) => {
 })
 
 test('drive.close()', async (t) => {
-  t.plan(2)
+  t.plan(4)
   const { drive } = await testenv(t.teardown)
   const blobs = await drive.getBlobs()
   blobs.core.on('close', () => t.ok(true))
   drive.core.on('close', () => t.ok(true))
+  drive.db.core.on('close', () => t.ok(true))
```

Comment:
`drive.core` is the same as `drive.db.core` (repeated listener)

Reply:
You're right, thanks! Fixed both occasions